ReplayMessenger.device.lib.nut

```diff
@@ -93,6 +109,41 @@ class ReplayMessenger {
         return _mm.send(messageName, data, null, _retryInterval, metadata);
     }
 
+    function onReplayBegin(loggerName, callback = null) {
+      if (callback == null && typeof loggerName == "string") { //de-register callback
+        if (loggerName in _onReplayBegin) {
+          delete _onReplayBegin[loggerName]
+        }
+      } else if (callback == null) { //assume callback is provided for "default" logger
```

Comment:
The block:

```
} else if (callback == null) { //assume callback is provided for "default" logger
    callback = loggerName;
    loggerName = "default";
}
```

does nothing and makes no sense to me. It should either be removed or fixed. 

Or am I terribly missing anything?